### ISSUE TITLE
Add universal PageUP/DOWN  binds

### ISSUE THREAD

**Issue by oy** [teeworlds/teeworlds#2202]
Add universal PageUP/DOWN binds that can be used to trigger ingame elements like chat scoreboard...

**Comment by Dune-jr** [comment]
Should they be used in the console as well?

What about switching from a spectator to another?

**Comment by oy** [comment]
Yeah could be used for the console as well. Not sure about spectators. You work with the mouse there and mousewheelUP/DOWN seem like a good choice.

**Comment by Dune-jr** [comment]
There are `spectate_next` and `spectate_previous` binds though, independently of the mouse

**Comment by oy** [comment]
Yeah, thought the default binds for them was mousewheelUp/DOWN, seems it's right/left key.
Not sure if people might think it's broken when it suddenly changes to pageUP/DOWN.

**Comment by Dune-jr** [comment]
It seems difficult to use the bind system to pilot the console, and the chat when `m_Chat` is not `CHAT_NONE`. Maybe `CBindsSpecial` could help with that actually? :)

**Comment by oy** [comment]
Special binds are reserved to Fx keys currently.
Think the LineInput which is used by the chat and console should drop keys it didn't handle like pageup/down, then binds should work. Something has to be done there.

**Comment by Dune-jr** [comment]
> Special binds are reserved to Fx keys currently.

We could extend special binds to those keys, since they could be used in the menus and what not.

> Think the LineInput which is used by the chat and console should drop keys it didn't handle like pageup/down, then binds should work. Something has to be done there.

This would mean that all sorts of binds could be triggered while in chat.

**Comment by oy** [comment]
Just keys that don't create characters or manipulate the chat could be triggered.
